README.md

```diff
@@ -1,84 +1,182 @@
 
-```
-poetry install
-```
+  *  Note: It is not mandatory that you need to download/use any of the pre-existing/already defined JSON mapper files that are present in this repository. You can always define your own custom JSON mapper files that follow a format similar to the ones defined in this repo.*
+
+  Create a directory for your mapper files and download them from this repository:
+  ```bash
+  mkdir input-files
+  cd input-files
+  ```
+
+  Download the mapper files you need from the [input-files directory](https://github.com/microbiomedata/metadata-for-user-facility-template-transformations/tree/main/input-files):
+    - For EMSL: `emsl_header.json`
+    - For JGI Metagenome: `jgi_mg_header.json` or `jgi_mg_header_v15.json`
+    - For JGI Metatranscriptome: `jgi_mt_header.json` or `jgi_mt_header_v15.json`
+
+1. **Set up your API access token**
 
-This will install the `mutts` package and create a `mutts` command-line tool.
+  Create a `.env` file in your working directory:
+  ```bash
+  echo "DATA_PORTAL_REFRESH_TOKEN=your_token_here" > .env
+  ```
 
-3. You need to obtain your NMDC Data and Submission Portal API Access Token and copy it over into your `.env` file, and associate it with the `DATA_PORTAL_REFRESH_TOKEN` environment variable. 
-   1. You can retrieve your Access Token by following this link: https://data.microbiomedata.org/user
-   2. Go over to the `.env` file and copy the Refresh Token like `DATA_PORTAL_REFRESH_TOKEN={refresh_token_value}`
+  To get your access token:
+  1. Visit https://data.microbiomedata.org/user
+  2. Copy your Refresh Token
+  3. Replace `your_token_here` in the `.env` file with your token
 
-4. Run the `mutts` command with options as follows:
+### Usage
+
+Run the `mutts` command with the required options:
 
 ```bash
 mutts --help
-Usage: mutts [OPTIONS]
-
-  Command-line interface for creating a spreadsheet based on metadata records.
-
-  :param submission: The ID of the metadata submission. 
-  :param user_facility: The user facility to retrieve data from. 
-  :param header: True if the headers should be included, False otherwise. 
-  :param mapper: Path to the JSON mapper specifying column mappings.
-  :param unique_field: Unique field to identify the metadata records. 
-  :param output: Path to the output XLSX file.
-
-Options:
-  -s, --submission TEXT       Metadata submission id.  [required]
-  -u, --user-facility TEXT    User facility to send data to.  [required]
-  -h, --header / --no-header  [default: no-header]
-  -m, --mapper PATH           Path to user facility specific JSON file.
-                              [required]
-  -uf, --unique-field TEXT    Unique field to identify the metadata records.
-                              [required]
-  -o, --output TEXT           Path to result output XLSX file.  [required]
-  --help                      Show this message and exit.
 ```
 
+#### Example 1: Generate a JGI Metagenome spreadsheet
+```bash
+mutts --submission <submission-uuid> \
```

Comment:
not sure where best to add this, but maybe document that the submission UUID is in the URL?